default/stringtables/es.txt

```diff
@@ -1173,29 +1173,29 @@ Escalado de coste de partes
 #*RULE_STARLANES_EVERYWHERE_DESC
 #* <not translated 1970-01-01 01:00:00>
 
-#*RULE_HABITABLE_SIZE_TINY
-#* <not translated 1970-01-01 01:00:00>
+RULE_HABITABLE_SIZE_TINY
+Hábitats en planetas enanos
```

Comment:
Corrección: **Espacio habitable** en planetas enanos
(Y lo mismo para el resto de tamaños).

Reply:
Nuevamente el problema es que el texto de la traducción literal no cabía en la ventana, por lo que cambié a hábitats, no se si hay alguna otra opción mejor...

Reply:
En ese caso, "Habitabilidad de planetas enanos". Si no caben ninguna de las dos, la versión completa, y espero que redimensionando la ventana se vea entero. In extremis, abrir un issue para que traten el tema de las ventanas demasiado pequeñas para algunos idiomas.

Reply:
De acuerdo, ya he subido el cambio, va a ser necesario redimensionar la ventana, por eso he puesto también al final lo de títlulo de ventanas.

Voy a crear la issue sobre los tamaños de ventana

Reply:
Entendido. Gracias por los screenshots. Yo creo la issue (primero abriré hilo en el foro).